app/jobs/reports/protocols_report.rb

```diff
@@ -13,10 +13,10 @@ def initialize(report_date = nil, *args, **rest)
       super(*args, **rest)
     end
 
-    def perform(report_date)
+    def perform(date = Time.zone.yesterday.end_of_day)
       return unless IdentityConfig.store.s3_reports_enabled
 
-      self.report_date = report_date
```

Comment:
I'd remove the initializer just in case and use the `attr_accessor` for specs

Reply:
the `ReportMailerPreview` needs the date added via initializer, since it bypasses the `perform` method  :(

Reply:
I think we could do
```
    report = Reports::ProtocolsReport.new.tap { |r| r.report_date = date }
```

Reply:
sure

Reply:
updated in https://github.com/18F/identity-idp/pull/11406/commits/42711be5ff60a042842f776a6c63533886a18eb1